buildSrc/build.gradle.kts

```diff
@@ -207,6 +207,7 @@ fun Project.applyGroovyProjectConventions() {
     tasks.withType<GroovyCompile>().configureEach {
         groovyOptions.apply {
             encoding = "utf-8"
+            forkOptions.jvmArgs?.add("-XX:+HeapDumpOnOutOfMemoryError")
```

Comment:
I suppose we don't need the heap dump in buildSrc, right? We can leave the arguments there, though. Any idea where the heap dump will end up?

Reply:
Heap dump of worker daemo will be in `~/.gradle/worker` and heap dump of buildSrc (Gradle daemon) will be in `~/.gradle/daemon`.